src/main/PdfLittleSigner/PdfSigner.cs

```diff
@@ -141,23 +153,44 @@ private IExternalSignature CreateExternalSignature(X509Certificate2 certificate)
                 .SetContact(iSignContact)
                 .SetLocation(iSignLocation);
 
-
             if (visible)
             {
-                signatureAppearance.SetPageRect(new iText.Kernel.Geom.Rectangle(_imageLocation.Width,
-                    _imageLocation.Height, _imageSize.Width, _imageSize.Height));
+                var signatureLocationX = pageSize.GetWidth() - _imageSize.Width - _stampMargin;
+                var signatureLocationY = pageSize.GetHeight() - _imageSize.Height - _stampMargin;
+                signatureAppearance.SetPageRect(new Rectangle(signatureLocationX, signatureLocationY, _imageSize.Width, _imageSize.Height));
 
                 if (stampFile != null)
                 {
+                    var stampImageExtension = Path.GetExtension(stampFile.FileName);
+                    if (Array.Exists(_unsupportedImageFormats, element => element.Equals(stampImageExtension)))
```

Comment:
zla implementacj
napisz:
```
_unsupportedImageFormats.Any(x=>x.ToLower() == stampImageExtension.ToLower())
```

Reply:
zamienione na
```csharp
var stampImageExtension = Path.GetExtension(stampFile.FileName);
if (stampImageExtension.ToLower().Equals(".png"))
```
zamiast odrzucać pliki png to są konwertowane na jpg a przezroczyste tło zamieniane na białe